tao_compiler/mlir/disc/tests/tensorflow_ops/bucketize.cc

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/compiler/mlir/disc/tests/mlir_feature_test.h"
+#include "tensorflow/compiler/mlir/disc/tests/mlir_test.h"
+#include "tensorflow/core/platform/test.h"
+
+namespace mlir_test {
+
+const std::string c_ft_path =
+    "tensorflow/compiler/mlir/disc/tests/tensorflow_ops/data/";
+
+// TODO(lanbo.llb): Support more input data types
+
+// f32 static shape test case
+TEST(TFBucketizeOpTest, StaticShapeF32) {
+  EXPECT_TRUE(feature_test_main(
+      /*mlir_file_path*/ c_ft_path + "bucketize_s_f32.mlir",
+      /*backend_types*/ { BackendType::kX86 },
```

Comment:
For BuketizeOp  itself,  it should be ok on ARM backend or cuda backend as well?  maybe we can add these backends as well.

Reply:
Currently `BuketizeOp` support cuda backend, should be added. For arm, I haven't done some tests on arm, but the op should support arm as well.

Reply:
modified to kSupportedBackend